src/collection-view.js

```diff
@@ -690,9 +691,14 @@ const CollectionView = Backbone.View.extend({
     return false;
   },
 
+  // Override to handle DOM inserting differently
+  beforeEl(el, siblings) {
+    this.$(el).before(siblings);
```

Comment:
This is good.

Although will not work with [Native.View](https://github.com/akre54/Backbone.NativeView/blob/master/backbone.nativeview.js#L69) but is something to be thought later